pkg/wireguard/agent/cell_test.go

```diff
@@ -0,0 +1,314 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
CI Build Commit taking too long to run and failing.
If it's essential it passes or if reviewer believe I can squash some commits please let me know.

Reply:
I vote for keep the individual commits around (assuming each commit itself builds) over pleasing CI and loosing the git history.